src/com/mie/controller/PostServlet.java

```diff
@@ -0,0 +1,81 @@
+package com.mie.controller;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import java.sql.Timestamp;
+
+
+import java.util.List;
+
+import com.mie.model.*;
+import com.mie.dao.*;
+
+public class PostServlet extends HttpServlet {
+	
+	private static final long serialVersionUID = 1L;
+	private static String POSTS = "/groups.jsp?group_id=";
+	
+	private PostDao dao;
+	
+	public PostServlet() {
+		super();
+		dao = new PostDao();
+	}
+	
+	public void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, java.io.IOException {
+		
+		String forward = "";
+		String action = request.getParameter("action");
+		
+		// start session
+		HttpSession session = request.getSession();
+		int postId = Integer.parseInt((String) session.getAttribute("postId"));
+		
+		if (action.equalsIgnoreCase("delete")) {
+			dao.deletePost(postId);
+			forward = POSTS; //ask kyle for the name of the page 
+			request.setAttribute("posts", dao.getAllPosts());
+			
+		} else if (action.equalsIgnoreCase("create")) {
+			Post post = new Post();
+			post.setPostId(createPostId());
```

Comment:
the ID column in Post should be set to autonumber so when you insert values into the other columns an ID is auto generated for it, don't need to set a post ID -- you can remove this line